src/App.jsx

```diff
@@ -1,3 +1,16 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { MovieList } from './pages/MovieList.jsx'
+import { ShowMovie } from './pages/ShowMovie.jsx'
+
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+  return (
+    <BrowserRouter>
+      <main>
+        <Routes>
+          <Route path="/" element={<MovieList />} />
+          <Route path="/movies/:id" element={<ShowMovie/>} />
+        </Routes>
+      </main>
+    </BrowserRouter>
+  )
 };
```

Comment:
Nice and clean App structure! Make sure you're consistent with double or single quotation marks